dpnp/dpnp_iface_types.py

```diff
@@ -140,6 +146,78 @@
 PZERO = numpy.PZERO
 
 
+def finfo(dtype):
```

Comment:
Could you add tests for these new functions(`finfo()`, `isdtype()`, `iinfo()`) to increase test coverage ?